code/modules/fishing/fish/_fish.dm

```diff
@@ -207,6 +207,9 @@
 	if(HAS_TRAIT(source, TRAIT_CATCH_AND_RELEASE))
 		context[SCREENTIP_CONTEXT_RMB] = "Release"
 		return CONTEXTUAL_SCREENTIP_SET
+	if(istype(held_item, /obj/item/clothing/neck/stethoscope))
+		context[SCREENTIP_CONTEXT_LMB] = "Check Pulse"
+		return CONTEXTUAL_SCREENTIP_SET
```

Comment:
This is in `add_item_context`, which is for when src is the held item and hovering another object. Move this to `add_context`